tests/bgp/test_bgp_chasssis.py

```diff
@@ -0,0 +1,424 @@
+import logging
+
+import copy
+import ipaddress
+import pytest
+import requests
+
+from natsort import natsorted
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.device_utils import fanout_switch_port_lookup
+from tests.voq.voq_helpers import get_vm_with_ip
+from tests.common.utilities import wait_until
+
+pytestmark = [
+    pytest.mark.topology('t2'),
+]
+
+logger = logging.getLogger(__name__)
+
+EXABGP_BASE_PORT = 5000
+EXABGP_BASE_PORT_V6 = 6000
+DUMMY_ASN1 = 64101
+IPV4 = 4
+IPV6 = 6
+IFUP = 1
+IFDOWN = 0
+
+def update_route(action, ptfip, port, route, asn=None):
```

Comment:
can you move APIs not specific to chassis, or that potentially be used by other test cases into "bgp_helpers.py".
This API can be moved there. Looks like "test_bgp_allow_list.py" has  the same API.